### PR TITLE
Add faction description tooltips to TD

### LINKED ISSUE #6288: Add faction tooltips to TD and D2K.
A little widget, when the faction is selected, or at least while the cursor is hovering over the name when choosing, that describes the faction.

example: a blurb like below would appear in a small window when deciding on the GDI team...

The GDI (Global Defense Initiative) is the international pact that aims to bring down the Brotherhood of Nod. They have technological prowess on their side.

An optional blurb, maybe with \n separation in the same text block:

Advantages: +3 attack, +2 move
Disadvantages: Higher cost

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/3938743-add-faction-tooltips-to-td-and-d2k?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by penev92**
Closes  #6288.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Looks good to me :+1: / :white_check_mark:

File: mods/cnc/rules/world.yaml [hunk 1]
```diff
@@ -21,12 +21,15 @@ World:
 		Name: Any
 		Race: Random
 		RandomRaceMembers: gdi, nod
+		Description: Select a random faction.
 	Country@gdi:
 		Name: GDI
 		Race: gdi
+		Description: Global Defense Initiative\nThe GDI are an international military branch of the United Nations tasked\nwith keeping world peace. Commanding the combined forces of the world's\nmost powerful nations, they possess an unmatched arsenal of high-tech weaponry.
```
Comment (obrakmann) [inline mods/cnc/rules/world.yaml]: "The GDI is"

**Comment by obrakmann** [comment]
:+1: otherwise

**Comment by penev92** [comment]
Fixed.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/ffc56c8d1183632060de5b6fd96eb09c3c8dcb49%5E...ffc56c8d1183632060de5b6fd96eb09c3c8dcb49)

**Pull request merged**